aries-site/src/examples/templates/popover/PopoverInlineExample.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useRef, useContext } from 'react';
+import { Popover } from 'aries-core';
+import {
+  Anchor,
+  Grid,
+  Card,
+  CardBody,
+  Button,
+  Box,
+  Paragraph,
+  Text,
+  Heading,
+  ResponsiveContext,
+} from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+
+const device = [
+  {
+    title: 'Require Assignments',
+    devices: '10',
+  },
+  {
+    title: 'Require Subscriptions',
+    devices: '10',
+  },
+  {
+    title: 'Assigned & Subscribed',
+    devices: '0',
+  },
+  {
+    title: 'Total Devices',
+    devices: '10',
+  },
+];
+
+const columns = {
+  xsmall: ['auto'],
+  small: ['auto'],
+  medium: ['auto'],
+  large: ['auto', 'auto', 'auto'],
+  xlarge: ['auto', 'auto', 'auto', 'auto'],
```

Comment:
Please revisit these values. I think we can probably avoid going to a single column until xsmall breakpoint.

Reply:
done